spec/javascripts/analytics-ga4/ga4-finder-tracker.spec.js

```diff
@@ -97,6 +97,7 @@ describe('GA4 finder change tracker', function () {
   it('creates the correct GA4 object for adding/removing a checkbox filter', function () {
     inputParent = document.createElement('div')
     inputParent.setAttribute('data-ga4-change-category', 'update-filter checkbox')
+    inputParent.setAttribute('data-ga4-filter-parent', '')
     inputParent.setAttribute('data-ga4-section', 'Your favourite chocolate')
```

Comment:
Sorry for the confusion, it's because having `data-ga4-filter-parent` above in the DOM is mainly relevant where you want to share the same index values between two inputs. If not, it can be placed at the same level.

I've updated one of the tests to reflect what you suggested testing. Thanks

Reply:
Good stuff 👍